test/messages.test.ts

```diff
@@ -114,7 +114,7 @@ describe("Message errors", () => {
                 "@type": "Mail",
                 subject: "A Subject",
                 body: "A Body"
-            }
+            } as any
```

Comment:
Can you create a `ConnectorMail` here instead?

Reply:
I want to turn of ts checks here, all good.